easybuild/easyblocks/l/llvm.py

```diff
@@ -381,6 +385,9 @@ def __init__(self, *args, **kwargs):
         self.log.info("Final projects to build: %s", ', '.join(self.final_projects))
         self.log.info("Final runtimes to build: %s", ', '.join(self.final_runtimes))
 
+        # CMake options passed to each building stage.
```

Comment:
For consistency with above:
```suggestion
        # CMake options passed to each build stage.
```